pandas/tests/indexes/period/test_ops.py

```diff
@@ -290,8 +290,9 @@ def test_comp_nat(self):
                                pd.Period('2011-01-03')])
         right = pd.PeriodIndex([pd.NaT, pd.NaT, pd.Period('2011-01-03')])
 
-        for l, r in [(left, right), (left.asobject, right.asobject)]:
-            result = l == r
+        for l, r in [(left, right),
```

Comment:
can you name these ``lhs, rhs`` instead of ``l, r``  (to avoid future flake warning)

Reply:
ok.